swap_meet/vendor.py

```diff
@@ -1,2 +1,93 @@
+from typing import ItemsView
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    #Wave 1
+    ''' We are creating an inventory, which is an empty list, for storing,removing and adding items'''
```

Comment:
I love the way you organized the docstrings by wave to not clutter up the file but still give useful information!